src/components/favorites-page/favorite-card/favorites-page-favorite-card.jsx

```diff
@@ -26,7 +27,8 @@ const FavoriteCard = ({item}) => {
             </span>
           </div>
           <button className="place-card__bookmark-button place-card__bookmark-button--active button"
-            type="button">
+            type="button"
+            onClick={(_evt) => onFavoriteChange(item.id)}>
```

Comment:
unused variable declaration `_evt`
```suggestion
            onClick={() => onFavoriteChange(item.id)}>
```

Reply:
По правилам линтера если параметр функции начинается с `_`, то ошибки не будет